script/github-actions/validate-build-status.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-console */
+/* eslint-disable camelcase */
+const fetch = require('node-fetch');
```

Comment:
This file is the one currently in master for `content-build`. Changes discussed and being made will be transferred over from `content-build` repo